INSTALL.md

```diff
@@ -227,14 +244,15 @@ If you lose this URL, it can be found in the `Info_Output.log` file under the "E
 
 ##### Accessing Elasticsearch Kibana server
 
-> NOTE: Kibana is only deployed in the default 'dev' stage; if you want Kibana set up in other stages, like 'production', please remove `Condition: isDev` from [elasticsearch.yaml](./cloudformation/elasticsearch.yaml)
+> NOTE: Kibana is only deployed in the default 'dev' stage; if you want Kibana set up in other stages, like 'production', please remove `Condition: isDev` from [elasticsearch.yaml](./cloudformation/elasticsearch.yaml) if using serverless, or in the [elasticsearch.ts](./lib/elasticsearch.ts) file.
```

Comment:
Nitpick - add 'if using CDK at the end of the sentence' just to be more clear?

Reply:
Good callout, added!